honeybadgermpc/hbavss.py

```diff
@@ -182,3 +191,146 @@ def _get_dealer_msg(self, value):
             v = None if values is None else values[i]
             avss_tasks[i] = asyncio.create_task(self.avss(k*avss_id+i, v, dealer_id))
         return await asyncio.gather(*avss_tasks)
+
+
+class HbAvssBatch(HbAvss):
+    async def _process_avss_msg(self, avss_id, dealer_id, dispersal_msg):
+        tag = f"{dealer_id}-{avss_id}-AVSS"
```

Comment:
can you give it a separate tag prefix? `B-AVSS`?

Reply:
Resolved.